generated/nidcpower/nidcpower.proto

```diff
@@ -376,8 +376,8 @@ enum CalibrationCloseAction {
 enum ComplianceLimitSymmetry {
   option allow_alias = true;
   COMPLIANCE_LIMIT_SYMMETRY_UNSPECIFIED = 0;
-  COMPLIANCE_LIMIT_SYMMETRY_NIDCPOWER_VAL_SYMMETRIC = 0;
-  COMPLIANCE_LIMIT_SYMMETRY_NIDCPOWER_VAL_ASYMMETRIC = 1;
+  COMPLIANCE_LIMIT_SYMMETRY_NIDCPOWER_VAL_COMPLIANCE_LIMIT_SYMMETRY_SYMMETRIC = 0;
```

Comment:
This is quite long and redundant. Could provide some information as to why this change was done?

Reply:
It looks like it was updated to match then name in the header file (see [here](https://github.com/doshirohan/grpc-device/blob/features/ni-dcpower/imports/include/nidcpower.h#L457)). It's unfortunate the constant also has `COMPLIANCE_LIMIT_SYMMETRY` in the name since we also prepend the name here with the enum name.

I think the same is done for the PowerAllocationMode enum a few lines down (476), the enum value names are taken straight from the header file. That being said this does look redundant and maybe for these in enums.py we could leave out `COMPLIANCE_LIMIT_SYMMETRY` and `POWER_ALLOCATION_MODE` from the enum values' names.

Reply:
This came out of discussion with Alex Dubois. You can find the discussion in [task description here](https://dev.azure.com/ni/DevCentral/_workitems/edit/1445386), but essentially we noticed that attribute values in the header file are named like this.
![image](https://user-images.githubusercontent.com/7554740/117157281-1912b980-addc-11eb-8ea3-e157082138b9.png)

But our metadata refers to them as VAL_SYMMETRIC and VAL_ASYMMETRIC. Alex said that we should be using descriptive names. I didn't realize that we also prefix the value with enum name making it quite long. I think we can revert this part of the change and CHANGES.md.

Reply:
@reckenro definitely makes sense. We will remove this change, and fixup and document the `POWER_ALLOCATION_MODE` enum values.

Palash, make sure you update PR description as well to reflect this change.

Reply:
Done and updated the PR description. Will close this PR once build completes.